Britannia/localization/english/BTWK_buildings_l_english.yml

```diff
@@ -49,7 +49,7 @@
  building_btwk_salt_mines_04_desc:0 ""
 
  building_type_btwk_lead_mines_01:0 "Lead Mines"
- building_type_btwk_lead_mines_01_desc:0 "Lead was a crucial metal to sustain irrigation infrastructure (despite its toxicity) during this period, as its malleability allowed for easy manipulation into constructs like pipes."
+ building_type_btwk_lead_mines_01_desc:0 "Lead's malleability allows for the easy manipulation into constructs like pipes, and was thus essential for maintaining Rome's irrigation infrastructure. While not as important now [try and tie in WRE collapse & the loss of skilled artisans], it still has its use in various construction works."
```

Comment:
While not as important now -> While the collapse of the empire and much of the Roman-era economy has significantly reduced demand for the metal, ?